migrations/0028-reharvest-automated-record-cards.js

```diff
@@ -0,0 +1,54 @@
+require('dotenv').config();
+
+const axios = require('axios');
+
+const recordOrigin = 'https://api.europeana.eu/record';
+const apiKey = process.env['EUROPEANA_RECORD_API_KEY'];
+
+const getRecord = async(identifier) => {
+  const url = recordOrigin + '/search.json?profile=minimal&query=europeana_id%3A%22' + identifier + '%22&rows=1';
+  return axios.get(url, {
+    params: {
+      wskey: apiKey
+    }
+  })
+    .then((response) => {
+      return response.data;
+    })
+    .catch((error) => {
+      throw error;
+    });
+};
+
+module.exports = async function(migration) {
+  migration.transformEntries({
```

Comment:
This thumbnail URL handling would only need to be done on the first run, right? As in, once we've done so, it won't be an issue in future, but we may still want to re-run the rest of the migration?

If so, then let's split this out into its own migration.